src/app/logic/Players/ComputerPlayer/ComputerPlayer.js

```diff
@@ -2,8 +2,9 @@ function createPlayer() {
     const player = {
         giveAnswerCallbacks: [],
 
-        onGiveAnswerDo(callback) {
-            this.giveAnswerCallbacks.push(callback);
+        onGiveAnswerDo(callbacks) {
+            const callbacksArray = Array.isArray(callbacks) ? callbacks : [callbacks];
+            this.giveAnswerCallbacks.push(...callbacksArray);
         },
```

Comment:
I think that onGiveAnswerDo method should throw Error if callbacks is not array of functions.

Reply:
✔️ done